src/pages/billing/CreateNewBillingProjectWizard.js

```diff
@@ -0,0 +1,367 @@
+import * as Auth from 'src/libs/auth'
+import colors from 'src/libs/colors'
+import { reportErrorAndRethrow } from 'src/libs/error'
+import { formHint, FormLabel } from 'src/libs/forms'
+import { getLocalPref, setLocalPref } from 'src/libs/prefs'
+import * as Utils from 'src/libs/utils'
+import { billingProjectNameValidator } from 'src/pages/billing/List'
+import validate from 'validate.js'
+
+
+export const styles = {
+  stepBanner: active => {
+    return {
+      borderRadius: '8px', boxSizing: 'border-box',
+      padding: '1.5rem 2rem',
+      marginTop: '1rem',
+      display: 'flex',
+      border: active ? '1px solid #426ead' : 'none',
+      backgroundColor: active ? '#4D72AA0C' : '#6D6E700D',
+      boxShadow: active ? '0 0 5px 0 rgba(77,114,170,0.5)' : 'none'
+    }
+  },
+  radioButtonLabel: {
+    marginLeft: '1rem', color: '#426ead', fontWeight: 500, lineHeight: '22px'
+  }
+}
+
+const CreateNewBillingProjectWizard = ({ onSuccess, billingAccounts, authorizeAndLoadAccounts, loadAccounts }) => {
+  const persistenceId = 'billing'
+  const [accessToBillingAccount, setAccessToBillingAccount] = useState(() => getLocalPref(persistenceId)?.accessToBillingAccount)
+  const [accessToAddBillingAccountUser, setAccessToAddBillingAccountUser] = useState(() => getLocalPref(persistenceId)?.accessToAddBillingAccountUser)
+  const [verified, setVerified] = useState(() => getLocalPref(persistenceId)?.verified || false)
+  const [billingProjectName, setBillingProjectName] = useState('')
+  const [chosenBillingAccount, setChosenBillingAccount] = useState('')
+  const [isBusy, setIsBusy] = useState(false)
+  const [existing, setExisting] = useState([])
+  const [billingProjectNameTouched, setBillingProjectNameTouched] = useState(false)
+  const [activeStep, setActiveStep] = useState(() => getLocalPref(persistenceId)?.activeStep || 1)
+
+  useEffect(() => {
+    setLocalPref(persistenceId, { activeStep, accessToBillingAccount, verified, accessToAddBillingAccountUser })
+  }, [persistenceId, activeStep, accessToBillingAccount, verified, accessToAddBillingAccountUser])
+
+  const next = () => {
+    setActiveStep(activeStep + 1)
+  }
+
+  const submit = _.flow(
+    reportErrorAndRethrow('Error creating billing project'),
+    Utils.withBusyState(setIsBusy)
+  )(async () => {
+    try {
+      await Ajax().Billing.createGCPProject(billingProjectName, chosenBillingAccount.accountName)
+      onSuccess(billingProjectName)
+    } catch (error) {
+      if (error.status === 409) {
+        setExisting(_.concat(billingProjectName, existing))
+      } else {
+        throw error
+      }
+    }
+  })
+
+  const step1 = () => {
+    const isActive = activeStep === 1
+    const isDone = activeStep > 1
+
+    return li({ 'aria-current': isActive ? 'step' : false, style: { ...styles.stepBanner(isActive), flexDirection: 'row', alignItems: 'center', justifyContent: 'space-between' } }, [
+      span({ className: 'sr-only' }, ['Step 1, step 1 of 4']),
```

Comment:
I know that @gabedemattos called this out, but I don't think we want this extra screen-reader text. We automatically get the number of steps and which one you are in because we are using semantic HTML (ordered list items). So adding this screen-reader text duplicates the information.

This is what it will read out if we just delete line 76.

<img width="634" alt="image" src="https://user-images.githubusercontent.com/484484/198093384-1717344c-9a6a-47e4-8e4e-41a8f06e250c.png">

Reply:
As opposed to getting this, when we have both the screen-reader text and the native list support.

<img width="625" alt="image" src="https://user-images.githubusercontent.com/484484/198094058-570463ba-090d-4ca0-b464-ac5e4ba57b41.png">